ddtrace/internal/remoteconfig/worker.py

```diff
@@ -80,17 +80,20 @@ def enable(self):
                 return True
 
             self.start()
-            forksafe.register(self.start_subscribers)
+            forksafe.register(self.reset_at_fork)
             atexit.register(self.disable)
             return True
         return False
 
-    def start_subscribers(self):
+    def reset_at_fork(self):
         # type: () -> None
-        """Subscribers need to be restarted when application forks"""
+        """Client Id needs to be restarted when application forks"""
         self._enable = False
         log.debug("[%s][P: %s] Remote Config Poller fork. Starting Pubsub services", os.getpid(), os.getppid())
         self._client.renew_id()
+
+    def restart_subscribers(self):
+        # type: () -> None
```

Comment:
I don't think anything should ever call this. Because it should be responsibility of all the products making use of RC to determine whether they need to restart after a fork, there should not be an attempt to restart all the subscribers at once. So probably the best thing to do here is remove this method entirely